.github/workflows/release.yml

```diff
@@ -11,18 +11,21 @@ on:
 
 jobs:
   release:
+    name: 🚀 Release
     runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/fmf
+    permissions:
+      id-token: write  # For pypi-publish
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         if: ${{ github.event_name == 'release' }}
-      - uses: actions/checkout@v2
-        if: ${{ github.event_name == 'workflow_dispatch' }}
+      - uses: actions/checkout@v4
         with:
           ref: ${{ github.event.inputs.ref }}
```

Comment:
Do we need this?

Reply:
In practice, probably redundant because `fmf`'s wheel are tagged with `-any`, so we wouldn't need to update the pypi wheels with more wheels. I'm fine with dropping that interface if there are no objections to this. The makefile will still have the `publish` workflow included as a backup until everyone is comfortable that the release workflow is stable.

Reply:
+1 for drop.  
imho Makefile can probably also drop publish, as it was tested in tmt and if somebody needs to create a manual publish, they might as well do tagging, hatch commands directly.